src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -209,6 +209,28 @@ private void initFullAccess(String gsaHostname, String[] fullAccessHosts) {
             + "access to content: {0}", new Object[] {addressesAndRanges});
   }
 
+  private boolean requestIsFromFullAccess(HttpExchange ex) {
+    InetAddress addr = ex.getRemoteAddress().getAddress();
+    return addressIsInFullAccess(addr);
+  }
+
+  private boolean addressIsInFullAccess(InetAddress addr) {
```

Comment:
So this method and requestIsFromFullAccess are not used anymore, yes? So can be deleted?